grc/blocks/variable_function_probe.block.yml

```diff
@@ -35,7 +35,13 @@ templates:
         def _${id}_probe():
           while True:
             <% obj = 'self' + ('.' + block_id if block_id else '') %>
-            val = ${obj}.${function_name}(${function_args})
+            <%
+            if function_args:
+                function_args_impr = ','.join(list('self.' + function_arg.strip() for function_arg in function_args.split(',')))
```

Comment:
Is it the case that any function args used to a method that would be used in the function probe would be `self. ...`?  What if the argument is a number or enum?

Reply:
Regardless of the type, wouldn't the argument still come in the form of a variable block? A variable block still includes the `self.` e.g.
```
        ##################################################
        # Variables
        ##################################################
        self.test = test = 5
        self.samp_rate = samp_rate = 192000
```
Are there cases where arguments come without a variable block?

Reply:
It wouldn't have to - what if my probe block was going to call the block method `blk.get_status(3)` - where 3 the value of some enum that returns what type of status to get.  Admittedly not the best design, but something someone could previously do using the function probe block

Reply:
Yes, that is a much clearer way to phrase it :)

Reply:
Yeah I think my PR would break arguments that are literal constants. Lemme work on it and ill mark it for review when Im done. Thanks for raising it josh